contrib/macdeploy/macdeployqtplus

```diff
@@ -16,9 +16,13 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-import subprocess, sys, re, os, shutil, stat, os.path, time
-from string import Template
+import plistlib
```

Comment:
This import was only used by the `-fancy` option and could also be removed since it is now unused.

Reply:
Good catch. However I'm going to merge this now, and we'll take care of the leftover import shortly.